staging/socket/examples/udp_write.lua

```diff
@@ -0,0 +1,8 @@
+local socket = require'socket'
+
+local s = socket.new("inet4", "dgram")
+
+s:connect("192.168.33.1", 8080)
+print("written: " .. s:write("alo"))
+
+s:close(s)
```

Comment:
@lneto s:connect could return the socket so we could do things like `local s = socket.new():connect()'; what do you think?

Also, on the other end there's just a simple UDP server -- socket, bind, read (user space).